azure-devops/azext_devops/test/team/test_team.py

```diff
@@ -0,0 +1,122 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+
+try:
+    # Attempt to load mock (works on Python 3.3 and above)
+    from unittest.mock import patch
+except ImportError:
+    # Attempt to load mock (works on Python version below 3.3)
+    from mock import patch
+
+from vsts.core.v4_0.core_client import CoreClient
+from azext_devops.dev.team.team import *
```

Comment:
From [pep8 guidelines](https://www.python.org/dev/peps/pep-0008/#imports) - _Wildcard imports (from <module> import *) should be avoided, as they make it unclear which names are present in the namespace, confusing both readers and many automated tools._

Reply:
I remember leaving the comment earlier to use wildcard as all the functions of the class were being imported. Looking at the doc shared by Atul, apparently wildcard is not recommended, so let's move back to using explicit imports instead of wildcard usage.

Reply:
Done.